`kraft cloud deploy` against Kraftfile panics

### Describe the bug

At the following line, `deployer` is `nil` when I run `kraft cloud deploy` without argument, against a Kraftfile like the one seen in #1490: 

https://github.com/unikraft/kraftkit/blob/6371181ba71f2625e590491b700dcd354242c978/internal/cli/kraft/cloud/deploy/deployer_kraftfile_runtime.go#L141

This leads to a panic:

```
 T  found digest=sha256:93d19ab0732b1c1c5efcd3eb2a1ff143dd76db85d414744f6f0ef32934c2be8b ref=index.unikraft.io/acotten.unikraft.io/myapp:latest
 i  found index.unikraft.io/acotten.unikraft.io/myapp:latest (kraftcloud/x86_64) (93d19ab, 699b766, 509 kB)
 D  found 1/1 matching packages in oci catalog
 T  checking compatibility packager=kraftfile-unikraft
 D  using packager=kraftfile-unikraft
[+] packaging myapp (oci)... done!                                                                                                                                                                                  x86_64/kraftcloud [0.0s]
[+] pushing index.unikraft.io/acotten.unikraft.io/myapp:latest (kraftcloud/x86_64)... done!    [2.0s]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x561daed9162d]
```

--- a/internal/cli/kraft/cloud/deploy/deployer_kraftfile_unikraft.go
+++ b/internal/cli/kraft/cloud/deploy/deployer_kraftfile_unikraft.go
@@ -69,5 +69,5 @@ func (deployer *deployerKraftfileUnikraft) Deploy(ctx context.Context, opts *Dep
 	}
 
 	// Re-use the runtime deployer, which also handles packaging.
-	return (*deployerKraftfileRuntime)(nil).Deploy(ctx, opts, args...)
+	return (&deployerKraftfileRuntime{}).Deploy(ctx, opts, args...)
 }

